JoelFaldin/ServerSort-MuniCrud-Backend
# Feat/add main routehandlers

# Add main routehandlers

## Description 🗒️✏️

Added route handlers for the tables `users`, `directions` and `departments`. Also added the prisma client for these tables.

## How to test it? 🎯⚗️🔬

- Install the dependencies using `pnpm i`.
- Start the project in development mode using `pnpm run start:dev`.
- For prisma, run `pnpx prisma migrate dev --name init`, then run `pnpx prisma generate`. Aditionally, you can start a local page to inspect the tables using `pnpx prisma studio`.
- Make GET requests to http://localhost:3000/api/${table-name}. The table names are `user`, `directions` and `departments`.
- Make POST requests to http://localhost:3000/api/${table-name} with the same table names from before.
- Make PATCH requests to http://localhost:3000/api/${table-name} with the table names. You can pass only the properties you want there.
- Make DELETE requests to the same address. Everything should be working correctly.
